llvm/lib/Transforms/Vectorize/VPlanRecipes.cpp

```diff
@@ -270,8 +270,6 @@ void VPRecipeBase::moveBefore(VPBasicBlock &BB,
 static Instruction *getInstructionForCost(const VPRecipeBase *R) {
   if (auto *S = dyn_cast<VPSingleDefRecipe>(R))
     return dyn_cast_or_null<Instruction>(S->getUnderlyingValue());
-  if (auto *IG = dyn_cast<VPInterleaveRecipe>(R))
```

Comment:
Remove `VPinterleaveRecipe` here will prevent `-force-target-instr-cost` overwriting recipe cost that may be inconsistent to the legacy model.

Reply:
The code has been reworked in fa3258ecb8f18702bb45fa7f7c5c436be9e575cc and the changes here are gone.